sybil/region.py

```diff
@@ -77,12 +77,12 @@ def __repr__(self) -> str:
             self.start, self.end, self.evaluator
         )
 
-    def __lt__(self, other):
+    def __lt__(self, other: Any) -> bool:
```

Comment:
Is there an annotation we could add in place of the `Any` that would match the assert below?
(ie: `other` should be the same type as `self`)

Reply:
Sure - I can make this use `'Region'`.